show/main.py

```diff
@@ -1237,8 +1237,12 @@ def techsupport(since, verbose, force, yes):
         if yes:
             cmd += " -f"
         else:
-            if click.confirm('Proceed with any system interruption?', abort=True):
+            if click.confirm('Proceed with any system interruption for HW register dump?'):
```

Comment:
I believe you should replace "any" with "potential" to make it clearer. Am I understanding the intention correctly?